titus-server-master/src/main/java/com/netflix/titus/master/scheduler/DefaultSchedulingService.java

```diff
@@ -542,24 +540,26 @@ private void checkIfExitOnSchedError(String s) {
                 schedulingService.requestResourceStatus(resourceStatus ->
                         printFenzoStateDump(mapper, "resource status", resourceStatus, latch));
             } catch (TaskQueueException e) {
-                logger.error("Couldn't request state dump from Fenzo: " + e.getMessage(), e);
+                logger.error("Couldn't request state dump from Fenzo: {}", e.getMessage(), e);
             }
             try {
                 if (!latch.await(MAX_DELAY_MILLIS_BETWEEN_SCHEDULING_ITERATIONS * 3, TimeUnit.MILLISECONDS)) {
                     logger.error("Timeout waiting for Fenzo state dump");
                 }
             } catch (InterruptedException e) {
-                logger.error("Interrupted while waiting for Fenzo state dump: " + e.getMessage(), e);
+                logger.error("Interrupted while waiting for Fenzo state dump: {}", e.getMessage(), e);
             }
             System.exit(3);
         }
     }
 
     private void printFenzoStateDump(ObjectMapper mapper, String what, Object dump, CountDownLatch latch) {
         try {
-            logger.info("Fenzo state dump of " + what + ": " + mapper.writeValueAsString(dump));
+            if (logger.isInfoEnabled()) {
```

Comment:
remove guard?

Reply:
Done, and added a comment to that effect.